### DISCUSSION TITLE
本项目要如何开启BBR拥堵算法，Shortcut-FE之类的功能呢？

### DISCUSSION THREAD

**Discussion by liyuhaolol** [immortalwrt/immortalwrt#1059] [Q&A]
### Describe the bug

通过我不断的google，我查到了作者已经把luci-app-turboacc这个东西删掉了。那么他其中的功能，我要在哪里开启呢？还是说功能也一并删除掉了？比如Shortcut-FE和BBR拥堵算法，我知道Fullcone可以去防火墙里开启。那么剩下几个功能呢？还能开启么？

### ImmortalWrt version

1

### ImmortalWrt target/subtarget

1

### Device

windows

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
openwrt 有原生的 flow offloading（软件流量分载），不需要经常崩溃的 sfe。
家用路由器开启 bbr 只会降低网络体验，除非你要在自己家开流媒体服务器。

**Reply by Kisoul** [reply]
原生的 flow offloading（软件流量分载）需要特别编译进去，然后特别设置一下吗？

**Reply by laaArmy** [reply]
> 原生的 flow offloading（软件流量分载）需要特别编译进去，然后特别设置一下吗？

不需要
